### ISSUE TITLE
unibeautify:disable and enable flags don't work consistently inside blocks

### ISSUE THREAD

**Issue by szeck87** [Unibeautify/unibeautify#46]
This correctly [doesn't apply beautification](https://playground.unibeautify.com/#N4IgNghgdg5grhGBTAchAtkkAuEApCANwgGUBjAJwEsAHAFxABoQB7eqlqAZx1AGEAxLxAAjJBDh0qAMypIKPbAG0QABQpI6dAJ4ACACIzpIALoBfZnxIlhYiVNnzFK9Zp0GjTNRq1yKXvBIAWgAhcUkZbVMLECsANVtwhz9nbzc9Q2ljc0sWMAATADE4Lg4oRPsZFJwXH3dM7JiAUUCKiMcFGrStDM9mQNCkyOjmAHEKCBoACwBFABk25Kcu1195EZAACQAVAFkF7FA7durlbvq+-GCwyukonK29uYBqJoAlEMWq5bPVi6yNptoPkwEgRBBOodREMOqk-r0Af1rjD7jFAgB5FBfWErOoI4zMP5+ALI26opEADWxp1q6Q8iPOxOYTRIcyoUAYSMGZI2BGI1J+tJ69Ma-SIpEotAYUOOS0hQv+BMZ62ZrPZnKu3Pa5JAcyQXEURxhNPO+K8RPWDzZAEc4FR8gL5ZqbtqNrsIQBrfIsADu5RlxsFyv8D12JToEDIUywAduJoGLocOreHGlRrjQfhIo2JCsNljJ0zeOzhLqxIeJDio0dcOLDRzEANNdxdPrFZokbBEMN0IzTqzbZiJB9VBgzd+dc8D22VAjUCocHQuhIOlBy+jmh7su+-cnAOnI7HBbltdbU5i220NCQ5Go9HHCrNpbc5ZicSoXAQYGkLAoZBj6aFruZ77m+cAAb2QGnmsIYxBS+wPqaJbOiiGwUtsSDoDQkB0BB244hOIGNDEv6juyEBgBhAAe0ogAA9HRuhwPO+HaNg+QfhAIiggAOlA0jMWQUicLo0ZgGALAAOq-gUAAUACUujAHxZCcFweRIAAdBJMCyQA5JsSDiSwujSRQBR6fJADcfFmCAZhAA)

```javascript
// unibeautify:disable
function test() {
  console.log("test");
}
```

This [does apply beautification](https://playground.unibeautify.com/#N4IgNghgdg5grhGBTAchAtkkAuEApCANwgGUBjAJwEsAHAFxABoQB7eqlqAZx1AGEAxLxAAjJBDh0qAMypIKPbAG0QABQpI6dAJ4ACACIzpIALoBfZnxIlhYiVNnzFK9Zp0GjTNRq1yKXvBIAWgAhcUkZbVMLECsANVtwhz9nbzc9Q2ljc0sWMAATADE4Lg4oRPsZFJwXH3dM7JiAUUCKiMcFGrStDM9mQNCkyOjmAHEKCBoACwBFABk25Kcu1195EZAACQAVAFkF7FA7durlbvq+-GCwyukonK29uYBqJoAlEMWq5bPVi6yNptoPkwEgRBBOodREMOqk-r0Af1rjD7jFAgB5FBfWErOoI4zMP5+ALI26opEADWxp1q6Q8iPOxOYTRIcyoUAYSMGZI2BGI1J+tJ69Ma-SIpEotAYUOOS0hQv+BMZ62ZrPZnKu3Pa5JAcyQXEURxhNPO+K8RPWDzZAEc4FR8gL5ZqbtqNrsIQBrfIsADu5RlxsFyv8D12JToEDIUywAduJoGLocOreHGlRrjQfhIo2JCsNljJ0zeOzhLqxIeJDio0dcOLDRzEANNdxdPrFZokbBEMN0IzTqzbZiJB9VBgzd+dc8D22VAjUCocHQuhIOlBy+jmh7su+-cnAOnI7HBbltdbU5i220NCQ5Go9HHCrNpbc5ZicSoXAQYGkLAoZBj6aFruZ77m+cAAb2QGnmsIYxBS+wPqaJbOiiGwUtsSDoDQkB0BB244hOIGNDEv6juyEBgBhAAe0ogNIcBQGQUicLo0ZgGALAAOq-gUAAUACUujAAAOlAAD0Ym6AxVD4do2D5B+EAiKColkJwXB5EgAB0HEwLxADkmxIOxLC6NxFAFPp-EANyiWYIBmEAA)

```javascript
function test() {
  // unibeautify:disable
  console.log("test");
}
```

**Comment by Glavin001** [comment]
Related to https://github.com/Unibeautify/unibeautify/issues/47

**Comment by Glavin001** [comment]
Related to https://github.com/Unibeautify/unibeautify/issues/39

**Comment by Glavin001** [comment]
Published to v0.9.3
